torch/utils/model_dump/__main__.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
+import sys
+from . import main
+
+sys.exit(main(sys.argv))
```

Comment:
TIL: `__main__.py`
I've always seen `if __name__ == '__main__':` so far.

Reply:
Yeah.  I was getting an error when trying to use "python -m torch.utils.model_dump" that told me to use this instead.

Reply:
I think related to the fact that this is __init__.py instead of model_dump.py.  The distinction between a package and module in Python is mysterious to me.